### DISCUSSION TITLE
Mocking with MSW

### DISCUSSION THREAD

**Discussion by jessiesiat-vennbio** [quasarframework/quasar#14282] [General - Components / Directives / etc]
Can't make [MSW](https://mswjs.io) work with Quasar, anyone has an idea on how to make it work

**Comment by MilosPaunovic** [comment]
Share with us what have you tried.

**Comment by jessiesiat-vennbio** [comment]
sorry I did not share it in the first place. It is an `SSR` type of app.

first I added a boot file in the `quasar.config.js`

    boot: [
     {
        server: false, // run on client-side only!
        path: "msw-worker"
     },
     ...
    ]

In the `src/boot/msw-worker.js` file I setup the browser worker, nothing special just a straightforward setup based from [MSW docs](https://mswjs.io/docs/getting-started/mocks). 

then finally I ran the `msw init` cmd in `dist/ssr` directory

there is no cli build error but I got this browser console error: 

<img width="1418" alt="Screen Shot 2022-08-30 at 9 24 42 AM" src="https://user-images.githubusercontent.com/99642482/187327304-549d249d-6905-428f-97f8-edbe65f5fe25.png">

--------

using Quasar version `1.14.2`

**Comment by little3201** [comment]
wait the answer

**Reply by little3201** [reply]
#16564 is useful
